crates/api/src/wire.proto

```diff
@@ -0,0 +1,61 @@
+syntax = "proto3";
+
+package kitsune2.wire;
```

Comment:
Do we want to version it? I know that buf recommends versioning the packages as such:
```suggestion
package kitsune2.wire.v1;
```

Reply:
It's just a [recommendation from buf](https://buf.build/docs/lint/rules/#package_version_suffix). I understand the same as you for the design of protobuf but I've only ever used them in production with https://buf.build. I'm not sure if I'd recommend it though but it's just what I'm familiar with so thought I'd ask :-)

Reply:
I'd submit it's [possible](https://www.hmeid.com/blog/just-say-no-to-versioning) to add functionality without breaking previous clients / versioning the protocol.